explainer.md

```diff
@@ -22,6 +22,11 @@ We're proposing an API to allow Web pages to request mirroring themselves to a
 secondary display.  They will be allowed to request customization of the
 mirroring implementation in certain respects.
 
+A secondary display is primarily a device on the local network connected
+wirelessly through technologies such as Chromecast or AirPlay, but may also be
+physically connected, such as through HDMI. Devices outside the LAN (i.e. in a
```

Comment:
You might say 

> (i.e. accessible via the Internet)

since that's the terminology in the charter.

https://www.w3.org/2022/03/charter-secondscreen-wg.html#scope